android/src/toga_android/widgets/base.py

```diff
@@ -182,18 +182,13 @@ def remove_child(self, child):
     # TODO: consider calling requestLayout or forceLayout here
     # (https://github.com/beeware/toga/issues/1289#issuecomment-1453096034)
     def refresh(self):
-        intrinsic = self.interface.intrinsic
-        intrinsic.width = intrinsic.height = None
+        # Default values; may be overwritten by rehint().
+        self.interface.intrinsic.width = at_least(self.interface._MIN_WIDTH)
+        self.interface.intrinsic.height = at_least(self.interface._MIN_HEIGHT)
```

Comment:
No, because they're already in CSS pixels. The intention of this PR is that when sizes are assigned to intrinsic.width or intrinsic.height, they are already in CSS pixels as the interface layer expects, so they don't need to be scaled again.